README.md

```diff
@@ -96,12 +96,13 @@ From the quarry-bastion:
 `git checkout <branch>` If not deploying main
 `git-crypt unlock <path to key>`
 `bash deploy.sh`
-`mysql -uquarry -h <trove hostname created in last step> -p < schema.sql`
 In horizon point the web proxy at the new cluster.
```

Comment:
is this required in redeployments? Does a new k8s cluster get created? I'm expecting only the `Deployment` k8s object to change, so not sure why the endpoint would change.

Reply:
By 'deployment' I mean to describe the procedure to deploy the project. Perhaps I should make a separate section for updating. Which is mostly just "run deploy.sh" though I haven't been testing it that way, and the ansible might need updated a little to make sure that is working.